config/application.rb

```diff
@@ -170,8 +169,7 @@ class Application < Rails::Application
     # Catching Invalid JSON Parse Errors with Rack Middleware
     config.middleware.use Middleware::CatchJsonParseErrors
 
-    # Record request ID in logging MDC storage
-    config.middleware.insert_before Rails::Rack::Logger, Middleware::TaggedLogging
```

Comment:
This patch drops `TaggedLogging` middleware and incorporate everything into `SessionSafeLogging` one.